drivers/fpga/fpga_ice40.c

```diff
@@ -95,7 +95,7 @@ struct fpga_ice40_config {
 	volatile gpio_port_pins_t *set;
 	volatile gpio_port_pins_t *clear;
 	uint16_t mhz_delay_count;
-	uint8_t creset_delay_ns;
+	uint8_t creset_delay_us;
```

Comment:
Likely this will need to be updated to `uint16_t` based on `spi.dtsi` that you have here.